src/ResizeSensor.js

```diff
@@ -171,16 +183,16 @@
                 reset();
             };
 
-            var addEvent = function(el, name, cb) {
+            var addPassiveEvent = function(el, name, cb) {
                 if (el.attachEvent) {
                     el.attachEvent('on' + name, cb);
                 } else {
-                    el.addEventListener(name, cb);
+                    el.addEventListener(name, cb, passiveEvents);
```

Comment:
@marcj Turns out scroll event doesn't need to be passive because scroll rendering is async and cannot be prevented anyway. It was wheel event that suffers from the issue. Sorry I should've gone deeper before opening a pull request.